localstack/services/events/models.py

```diff
@@ -102,3 +106,20 @@ class InvalidEventPatternException(Exception):
     def __init__(self, reason=None, message=None) -> None:
         self.reason = reason
         self.message = message or f"Event pattern is not valid. Reason: {reason}"
+
+
+class FormattedEvent(BaseModel):
```

Comment:
It is quite different since the initial defined type was not accurate. We transform the incoming event (here: https://github.com/localstack/localstack/blob/feature/eventbridge_v2-add-input-path/localstack/services/events/provider.py#L129) and add new fields since they are expected by the client.